functions.php

```diff
@@ -50,6 +50,34 @@ function c4aa_add_body_class_from_acf_options( $classes ) {
 }
 add_filter( 'body_class', 'c4aa_add_body_class_from_acf_options' );
 
+/**
+ * Add Dashboard Widget
+ * 
+ * A place for specific reference info & instructions about the theme 
+ * for users of the site.
+ * 
+ */ 
+
+
+add_action('wp_dashboard_setup', 'c4aa_custom_dashboard_widgets');
+  
+function c4aa_custom_dashboard_widgets() {
+global $wp_meta_boxes;
+ 
+wp_add_dashboard_widget('custom_help_widget', 'C4AA Theme Notes', 'custom_dashboard_help');
+}
+ 
+function custom_dashboard_help() {
```

Comment:
@slambert Add c4aa_ name space to this function also, so `c4aa_dashboard_help`, then this is good!

Reply:
Done - thanks!